package.json

```diff
@@ -109,6 +109,7 @@
     "typescript": "^4.5.2"
   },
   "dependencies": {
+    "@codibre/fluent-iterable": "*",
     "@grpc/grpc-js": "^1.3.1",
     "@grpc/proto-loader": "^0.6.2",
     "grpc": "^1.24.11"
```

Comment:
I know that was not modified on this PR, but do we need grpc (and others libraries) as dependencies and peerDependencies at same time? Probally change that now will be a breaking change, but we can check it out for next major (whenever it cames out).